scripts/populate_db.py

```diff
@@ -1,6 +1,7 @@
 """This will populate the database from the google sheet."""
 from absl import app
 from absl import flags
+import click
```

Comment:
we would need to add this to the requirements.txt too.

however i am not even sure this script is used any more with the new db

Reply:
It is already installed because used by `scripts/wipe_sensitive_info_db.py`. Tell me if you want to delete this script.

Reply:
I think Romain is updating it, so let's leave it for now.